pkg/k8shandler/configmap.go

```diff
@@ -97,6 +97,10 @@ func (clusterRequest *ClusterLoggingRequest) RemoveConfigMap(configmapName strin
 		map[string]string{},
 	)
 
+	if !HasCLORef(configMap, clusterRequest) {
```

Comment:
can you quickly comment that we should remove this next release so it doesn't get left behind?

Reply:
fixed